readthedocs/settings/proxito/base.py

```diff
@@ -11,6 +11,7 @@ class CommunityProxitoSettingsMixin:
 
     ROOT_URLCONF = 'readthedocs.proxito.urls'
     USE_SUBDOMAIN = True
+    SECURE_REFERRER_POLICY = "no-referrer-when-downgrade"
```

Comment:
This makes sense 👍

Reply:
To be honest, I think `strict-origin-when-cross-origin` would be fine for docs sites as we don't really use the additional data as far as I'm aware but at least we have an easy upgrade path this way.